GitVersionConfig.yaml

```diff
@@ -1,4 +1,4 @@
-next-version: 2.1
+next-version: 2.2
```

Comment:
We shouldn't really need this, can you please remove it and see what version number we get?